kubechain/internal/humanlayer/hlclient.go

```diff
@@ -0,0 +1,47 @@
+// Note, may eventually move on from the client.go in this project
```

Comment:
This is _not_ generated with the OpenAPI generator and will be used in future PRs.